flexget/plugins/sites/piratebay.py

```diff
@@ -65,20 +64,23 @@ class UrlRewritePirateBay(object):
         ]
     }
 
+    url_match = re.compile('^%s://(?:torrents\.)?(%s)/.*$' % (re.escape(PROTOCOL), re.escape(DOMAIN)))
```

Comment:
Yes, but I wasn't sure if the config would have been initialized if the url-rewrite functions would be used. So that is why I initalize them here for the default thepiratebay.org-value